src/org/zaproxy/zap/extension/autoupdate/ExtensionAutoUpdate.java

```diff
@@ -634,34 +634,6 @@ public void alertIfNewVersions() {
     	
     	final OptionsParamCheckForUpdates options = getModel().getOptionsParam().getCheckForUpdatesParam();
     	
-		if (View.isInitialised()) {
-			if (options.isCheckOnStartUnset()) {
-				// First time in
-                int result = getView().showConfirmDialog(
-                		Constant.messages.getString("cfu.confirm.startCheck"));
-                if (result == JOptionPane.OK_OPTION) {
-                	options.setCheckOnStart(true);
-                	options.setCheckAddonUpdates(true);
-                	options.setDownloadNewRelease(true);
-                } else {
-                	options.setCheckOnStart(false);
-                }
-                // Save
-			    try {
-			    	this.getModel().getOptionsParam().getConfig().save();
-	            } catch (ConfigurationException ce) {
-	            	logger.error(ce.getMessage(), ce);
-	                getView().showWarningDialog(
-	                		Constant.messages.getString("cfu.confirm.error"));
```

Comment:
Can be removed as well? (Does not seem to be used in other places.)

Reply:
Yep, done.